be/src/olap/primary_key_index.cpp

```diff
@@ -70,6 +74,12 @@ Status PrimaryKeyIndexReader::parse(io::FileReaderSPtr file_reader,
     _index_reader.reset(new segment_v2::IndexedColumnReader(file_reader, meta.primary_key_index()));
     RETURN_IF_ERROR(_index_reader->load(!config::disable_storage_page_cache, false));
 
+    _parsed = true;
```

Comment:
the `_parsed` variable should separate for pk_index and bf?
e.g. in `check_present`, there is a `DCHECK(_parsed)`, then lookup bf, but `_parsed` can't guarantee the bf parsed.